agent.py

```diff
@@ -236,6 +237,12 @@ def main():
         sys.stderr.write("Unknown command: %s\n" % command)
         return 3
 
+    # Deprecation notice
+    if command not in DD_AGENT_COMMANDS:
+        # Will become an error message and exit after deprecation period
+        from utils.deprecations import deprecate_old_command_line_tools
+        deprecate_old_command_line_tools(__file__)
+
```

Comment:
Maybe this could be more granular as we're only deprecating certain subcommands and not `jmx, `check` and `flare` ?